newbank/server/BankUser.java

```diff
@@ -0,0 +1,10 @@
+package server;
```

Comment:
Just looking through the code, Noticed some of the package name is slightly different for the newer classes, Package server as oppose to package newbank.server. 
Not sure if this will affect how it runs.

Reply:
👍 updated